jenkins_job_wrecker/modules/builders.py

```diff
@@ -1,5 +1,7 @@
 # encoding=utf8
 import jenkins_job_wrecker.modules.base
+from distutils.util import strtobool
```

Comment:
I'd rather avoid a dependency on distutils. In other parts of code the `.text` is usually compared with 'true' to create a bool result, i.e.

    build_name["file"] = (build_name_element.text == 'true')

Reply:
Ack